Docker/entrypoint.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+
```

Comment:
It is also available in shell http://pubs.opengroup.org/onlinepubs/009695399/utilities/set.html